moveit_ros/visualization/motion_planning_rviz_plugin/include/moveit/motion_planning_rviz_plugin/motion_planning_frame.h

```diff
@@ -172,8 +172,8 @@ private Q_SLOTS:
   void onClearOctomapClicked();
 
   // Scene Objects tab
-  void importFileButtonClicked();
-  void importUrlButtonClicked();
+  void import3DObjectFromFileButtonClicked();
```

Comment:
IMO, you should revert renamings of all (internal) symbols. At most, I agree with import**From**[File|Url]...
If you want to clarify (for the user) that a single object (instead of a full scene) is imported, provide a tooltip to the corresponding buttons.

Reply:
I'd go for "Object" over "3DObject", but I would like to separate a saved Scene from a saved Object more clearly. Is there a reason not to be descriptive?

I think the fact that these functions are closely related and somewhat easy to confuse is that `computeLoadSceneButton` is defined in `motion_planning_frame_objects.cpp` instead of `motion_planning_frame_scenes.cpp` . I assume the separation was introduced later during development.

Reply:
I agree that we should distinguish scene and object loading/saving if needed. But can we actually import a whole scene?
If so, the corresponding symbols should be renamed to *_scene_* as well?
However, if there is only object loading, why should we rename the symbols?

Reply:
I think it's sufficiently messy to clear up. There are:  
- "import_file_button" which connects to `importFileButtonClicked` and imports a mesh
- "import_url_button" which connects to `importURLButtonClicked` and imports a mesh
- "import_scene_text_button" which connects to `importFromTextButtonClicked" and imports a .scene file (containing scene geometry (shapes), but no other info)
- "load_scene_button" which connects to `loadSceneButtonClicked` which loads a scene from `moveit_warehouse` (I don't know if this actually works)
- "load_state_button" which connects to `loadStateButtonClicked` and loads robot states (not necessarily clear to first-time codebase readers that this is only the robot state, not the state of the scene)
- "load_query_button" which loads planning queries, and a few corresponding "save" buttons.

The symbols are all in the same big `motion_planning_frame` file, so I think they should be clearer.

Reply:
As mentioned below, I renamed some more of these.